res/values/strings.xml

```diff
@@ -21,4 +21,7 @@
     <string name="quote_download_failed">Failed to download quote</string>
     <string name="quote_download_success">Quote downloaded successfully</string>
     <string name="module_vnaas_name">Visual Novel as a Service</string>
+    <string name="module_goodreads_name">Goodreads QOTD RSS</string>
+    <string name="module_hitokoto_name">Hitokoto 一言</string>
+    <string name="module_wikiquote_name">Wikiquote QOTD</string>
```

Comment:
Add a note that this is for Chinese (`Wikiquote QOTD (zh)` or something like that)